requirements/base.txt

```diff
@@ -22,10 +18,14 @@ jsonfield==1.0.3
 lazy==1.1
 libsass==0.10.0
 loremipsum==1.0.5
+lxml==3.7.3
+numpy==1.12.1
 path.py==8.2.1
 python-dateutil==2.1
 python-memcached==1.48
 pytz
+scikit-learn==0.18.1
+scipy==0.19.0
```

Comment:
This is the results of:

1.  install each entry previously in `wheels.txt` without a version specifier
2. see which version was installed, and pin that here

I had to do this as some of the previously checked-in wheels were for versions that are no longer available via `pip`.

I think quite a few of these are only needed by AI grading, we ought to excise that unused feature from ORA entirely at some point.